OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -2470,30 +2470,54 @@ public static DreamValue NativeProc_sorttext(NativeProc.State state)
             return new DreamValue(state.ResourceManager.SaveTextToFile(file, text) ? 1 : 0);
         }
 
+        private static int? StringToInteger(string value, int radix) {
+            if (value == null || value.Length == 0)
```

Comment:
```suggestion
            if (string.IsNullOrWhiteSpace(value))
```

Reply:
I changed "string" to "ReadOnlySpan", so the string library function is no more working here.